include/lsst/meas/base/CentroidUtilities.h

```diff
@@ -44,21 +44,21 @@ struct CentroidResult {
     CentroidResult();
 
     /// Constructor; initializes everything from values.
-    explicit CentroidResult(CentroidElement _x, CentroidElement _y, CentroidCov const & matrix):
-        x(_x),
-        y(_y),
+    explicit CentroidResult(CentroidElement x, CentroidElement y, CentroidCov const & matrix):
+        x(x),
+        y(y),
```

Comment:
I strongly disagree (clearly, else I would not have made the change). I personally hate arguments with needless underscores and in a situation like this where the arguments are being used to attributes that have the same name, using the same name seems intuitive and natural to me. I realize it can be a bit confusing to a newbie, but since the results are obvious I see that as harmless.